docs/guides/src/main/server/configuration-production.adoc

```diff
@@ -36,4 +36,19 @@ Keycloak runs on top of JGroups and Infinispan, which provide a reliable, high-a
 
 To find out more about using multiple nodes, the different caches and an appropriate stack for your environment, see the <@links.server id="caching"/> guide.
 
+== Stack and address preference
+
+The system properties `java.net.preferIPv4Stack` and `java.net.preferIPv6Addresses` are used to configure the JVM for use with IPv4 or IPv6 addresses.
+
+With Keycloak, in order to run using IPv4 addresses, you need to specify `java.net.preferIPv4Stack=true`; in order to run with IPv6 addresses,
```

Comment:
```suggestion
Keycloak uses the IPv4 address stack by default. In order to run with IPv6 addresses,
```

Reply:
@pedroigor This is the default, so there is not really a "need to specify" it, right?

Reply:
Changed.